addons/mail/static/src/models/thread/thread.js

```diff
@@ -1255,6 +1285,7 @@ function factory(dependencies) {
         followers: one2many('mail.follower', {
             inverse: 'followedThread',
         }),
+        suggestedPartners: attr(),
```

Comment:
We try to add comments above fields now.

Reply:
I do not really see what kind of comment I should write ?